internal/pkg/manifest/pipeline.go

```diff
@@ -44,7 +44,9 @@ func (p *githubProvider) Properties() map[string]interface{} {
 type GitHubProperties struct {
 	// use tag from https://godoc.org/github.com/fatih/structs#example-Map--Tags
 	// to specify the name of the field in the output properties
-	Repository string `structs:"repository" yaml:"repository"`
+
+	// An example for OwnerAndRepository would be: "aws/amazon-ecs-cli-v2"
+	OwnerAndRepository string `structs:"repository" yaml:"repository"`
```

Comment:
Is there a reasoning behind this rename? I kinda prefer `Repository` over `OwnerAndRepository`.

Reply:
In Cloudformation it explicitly distinguish between the user name "aws" and repository name "amazon-ecs-cli-v2". That's why I think being explicit here is better. For customers, we still only show "repository"